api/aperture/policy/language/v1/classifier.proto

```diff
@@ -15,17 +15,15 @@ import "protoc-gen-openapiv2/options/annotations.proto";
 //
 // Example:
 // ```yaml
-// selector:
-//   service_selector:
-//     service: service1.default.svc.cluster.local
-//   flow_selector:
-//     control_point:
-//       traffic: ingress
+// flow_selector:
+//   control_point:
```

Comment:
that is not a correct structure of flow_selector, please check properly
flow_selector has service_selector and flow_matcher with their respective fields